### ISSUE TITLE
Trakt marks?

### ISSUE THREAD

**Issue by JayDee696969** [jurialmunkey/plugin.video.themoviedb.helper#93]
Ok, whats the trick to get episodes marked as watched after watching them? I synced my Trakt account the TMDH. All my shows and movies import fine. After watching an episode I assumed TMDH would automatically mark the episode as watched, but it doesnt seem to do that. I also dont see an option to mark as watched in Trakt. Do I also need to have the Kodi Trakt addon and context menus installed along with TMDH? I have enabled the option to get watched indicators from Trakt, but it doesnt seem to automatically mark them after watching. Any advice? Thanks.

**Comment by drinfernoo** [comment]
That is all handled by the add-on you're playing through, and isn't handled at all by TheMovieDb Helper.

**Comment by JayDee696969** [comment]
> That is all handled by the add-on you're playing through, and isn't handled at all by TheMovieDb Helper.

Ohhh, well that kinda sucks. So your saying I would need to activate Trakt on every addon that i use with the players?? Wouldnt it make more sense to have TMDBH handle this instead? Or am i missing the point?

**Comment by JayDee696969** [comment]
Hmm, but I wonder if.... instead of using players that just open the addons scrapers, if TMDBH was able to import the scrapers of these addons instead and used itself to play the streams instead of the addons then it would be able to have a functioning Trakt correct?

**Comment by drinfernoo** [comment]
That's far out of the scope of this project, unless @jurialmunkey feels otherwise 🤷🏾‍♀️

**Comment by JayDee696969** [comment]
I hear ya, my fault. I was thinking of TMDBH like it was an addon that actually plays the link. I understand its more of a frontend to open the streams using other addons. I completely understand now why it doesnt work the way i thought. I was just super impressed with the speed, Trakt options, and other features it has.

**Comment by JayDee696969** [comment]
Maybe, just possibly he can add the option to import or use a scraper, like a4k or OpenScrapers? Then TMDBH can play the link, which means it could also capture watched status and report it to Trakt? I think? That would make my life complete.. LOL.

**Comment by JayDee696969** [comment]
Even better if you guys can merge OpenInfo script GUI with TMDBH. Seems like a match made in heaven to me HAHA. Ok, ill shutup now....maybe.

**Comment by Demonseeed** [comment]
I believe jurialmonkey wants this addon to stay in the Official Kodi Repo, and adding those scrapers would most definitely get it banned.

**Comment by JayDee696969** [comment]
For Forks sake!! Pun intended..

**Comment by JayDee696969** [comment]
Oh, and the Randomized section is awesome for widgets!!

**Comment by jurialmunkey** [comment]
I'm not interested in making a plugin that serves content - that's not the intent of the plugin. The idea is to provide one interface which joins the local library with content from other addons. I don't care what addons someone else uses, but there are plenty of legitimate legal uses (e.g. connecting with Netflix, Amazon, Youtube, PKC etc.).

I added the player functionality because I got annoyed with having my local library separated from my Netflix content. Now I'm able to have widgets that allow me to either play from local library, Netflix, or view trailers on Youtube.

There were lots of aspects of OpenMeta that I didn't like and it's bundling of banned addon players meant that it could never be integrated into a skin on the official repo. The primary purpose of TMDbHelper is to afford skinners the tools to create a more integrated interface - I would really hope that people will respect this purpose and contribute to the addon to make it better rather than forking it purely for piracy purposes.
